deltacat/compute/compactor_v2/compaction_session.py

```diff
@@ -666,6 +666,12 @@ def merge_input_provider(index, item):
         f"compacted at: {params.last_stream_position_to_compact},"
     )
     is_inplace_compacted: bool = (
+        params.rebase_source_partition_locator
```

Comment:
You can use `rcf_source_partition_locator`. Also, please add a test case that fails for this scenario.

Reply:
Sure. Does `rcf_source_partition_locator` replace only `params.rebase_source_partition_locator` or does it also replace using `params.source_partition_locator`?

Reply:
both. See how it's defined above.